mypy.ini

```diff
@@ -27,3 +27,6 @@ warn_return_any = true
 warn_unreachable = true
 warn_unused_configs = true
 warn_unused_ignores = true
+
+[mypy-rillrate.*]
+ignore_missing_imports = true
```

Comment:
Instead of ignoring globally, you can use `# type: ignore[import]` in the erroring line

Reply:
It may be good to leave it this way for the future, if RillRate is ever implemented on the library itself (which can be done without conflicts)
Single line ignores will get annoying if that becomes the case.

Reply:
Thats a good point, might be worth moving the other type ignores for imports here too (cc @FasterSpeeding)